setup.bat

```diff
@@ -1,5 +1,7 @@
 cd rg_speech_to_text
-python3.8 -m venv venv
+python -m venv venv
 rem venv\Scripts\activate : activating venv breaks the bat file execution
 venv\Scripts\python -m pip install --upgrade pip setuptools
```

Comment:
this line is not modified by your PR but I still wanted your review if it is needed inside the venv or if it is to run on the external user system environment

Reply:
for me it can find only simple python, the other one is missing.

Reply:
but what do you have python version ? when you type python --version ?
I prefer to keep it that way and it is up to the user to create a symlink or redirect python3.8 to the right python path
I'll think how to document that

Reply:
I'll take your pr and I'll think about this default python version issue

Reply:
thx.